usersimcrs/core/simulation_domain.py

```diff
@@ -0,0 +1,28 @@
+"""Simulation domain knowledge."""
```

Comment:
Can you expand this description to clarify how it differs from Domain?

Do we really need a separate class here instead of just extending Domain with requestable slots?

Reply:
Indeed, there are two approaches here:

1. Either we create a child class as I proposed, it allows us to define different types of slots.
2. We can decide to extend the original class `Domain` from DialogueKit. 

After second thoughts I would say that we can distinguish two types of slots, i.e., informable and requestable, in a general setting. So, I guess we can go for option two with a slightly different structure fro the YAML file:

```yaml
name: <domain_name>
informable_slots:
 - <slot1>
 - <slot2>
requestable_slots:
 - <slot3>
 - <slot4>
```

What do you think?